### ISSUE TITLE
Advanced person search problems

### ISSUE THREAD

**Issue by TheGrandVizier** [Beit-Hatfutsot/dbs-back#38]
Some of the advanced fields do not work, at all, producing either an infinite loader or a simple "no results found" error.

Basic:
Tree number alone, produces infinite loader.
Anything + Gender produces an "no results found" or infinite loader.

More advanced reproductions:

1. Search For Cohen as last name.
2. Add "Russia" as birthplace. So far, so good.
3. Add Tree serial number: 1115
Result: http://test.dbs.bh.org.il/person?last_name=cohen&birth_place=Russia&tree_number=1115&more=1
finds zero results or fails search

Alternate scenario 1:
1. Search for Cohen as last name
2. Refine to birthplace Russia.
3. Add female only.
Result: http://test.dbs.bh.org.il/person?last_name=cohen&sex=f&birth_place=Russia&more=1

Alternate 2:
1. Search for Cohen as last name.
2. Add Russia as birthplace.
3. Add gender "nevermind"
Result: http://test.dbs.bh.org.il/person?last_name=cohen&sex=n&more=1

To summarize:
Tree ID search doesnt work.
Gender doesn't work.
